sway-lib-std/src/tx.sw

```diff
@@ -42,6 +42,18 @@ const TX_WITNESSES_COUNT_OFFSET = 10312;
 const TX_RECEIPTS_ROOT_OFFSET = 10320;
 const TX_SCRIPT_START_OFFSET = 10352;
 
+// Input types
+const INPUT_COIN = 0u8;
+const INPUT_CONTRACT = 1u8;
+const INPUT_MESSAGE = 2u8;
+
+// Output types
+const OUTPUT_CONTRACT = 1u8;
```

Comment:
Hmm. Shouldn't there be an `OUTPUT_COIN` too?

Reply:
Ha, yes - definitely. Added